stories/s2l1c.stories.js

```diff
@@ -213,6 +216,36 @@ export const FindTiles = () => {
   return wrapperEl;
 };
 
+export const FindTilesCatalog = () => {
```

Comment:
This is weird - with `FindTilesSearchIndex` the call to `searchIndex` is made (as expected), but with `FindTilesCatalog` another endpoint, `search` is used... Am I missing something?